gradle/libs.versions.toml

```diff
@@ -94,11 +94,11 @@ androidx-arch-test-core = "androidx.arch.core:core-testing:2.1.0"
 
 androidx-compose-compiler = { module = "androidx.compose.compiler:compiler", version.ref = "androidx-compose-compiler" }
 
-androidx-compose-foundation = { module = "androidx.compose.foundation:foundation", version.ref = "androidx-compose-runtime" }
+androidx-compose-foundation = { module = "androidx.compose.foundation:foundation", version = "1.3.1" }
```

Comment:
TODO update to Compose BOM

Reply:
The way I see it, it's better to separate the changes - what if the library author doesn't want to switch to BOM?

Reply:
Going to put my reasoning here for the initial TODO in case the author would like me to upgrade to the Compose BOM as part of this PR (Or a follow up)

Before my changes the version catalog is using `androidx-compose-runtime` version for many different compose artifacts (foundation, material and runtime) but new updates to each of these libraries are no longer being versioned together. Updating to use the Compose BOM would enable just tracking a single BOM version and not worrying about each individual artifact.